deprecations/README.md

```diff
@@ -36,6 +36,8 @@ The table columns have the following meanings:
 
 | Feature | Comments | Deprecated | Discontinued |
 | :-- | :-- | :-- | :-- |
+| `BasePrices`<br>in [Get rate pricing](../operations/rates.md#get-rate-pricing) | Replaced by `BaseAmountPrices` | 26 Oct 2022 | 26 Apr 2022 |
```

Comment:
Why is the deprecation date in October?

Reply:
Good point, the deprecation date should of course be the same as the Changelog date. I'm going to update the date and get this PR merged.